server/package.json

```diff
@@ -29,7 +30,7 @@
         "tween-functions": "^1.2.0",
         "utf-8-validate": "^5.0.2",
         "uuid": "^8.3.1",
-        "forge-dataviz-iot-data-modules": "0.1.10"
+        "forge-dataviz-iot-data-modules": "/Users/t_agrap/Documents/forge-dataviz-iot-data-modules/forge-dataviz-iot-data-modules-0.1.11.tgz"
```

Comment:
You're committing your local path in the package.json

Reply:
Will resolve this.